src/fabric/src/fabric2_server.erl

```diff
@@ -142,3 +170,37 @@ get_env(Key) ->
         Value ->
             Value
     end.
+
+
+apply_tx_options(Db, Cfg) ->
+    maps:map(fun(Option, {Type, Default}) ->
+        case lists:keyfind(atom_to_list(Option), 1, Cfg) of
+            false ->
+                case Default of
+                    undefined -> ok;
```

Comment:
I think this is wrong. If you change one of these values to a non-default value, then we'd not be able to delete the config entry to revert to a default value. Though I'm wondering what that would be at the C level now that I think about it. Empty string? Or NULL maybe?

Reply:
good point, I don't think there is a easy to to reset the option. I wonder if it is safe to just replace the db handle